PhoenixPointModLoaderInjector/PhoenixPointModLoaderInjector.csproj

```
@@ -83,4 +83,11 @@
     </PackageReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>copy /Y "$(TargetPath)" "C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Managed\$(TargetFileName)"
+copy /Y "C:\Users\Ijwu\OneDrive\Documents\Code\PhoenixPointModInjector\PhoenixPointModLoader\bin\Debug\PhoenixPointModLoader.dll" "C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Managed\PhoenixPointModLoader.dll"
+copy /Y "C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Assembly-CSharp.dll.orig" "C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Managed\Assembly-CSharp.dll"
+cd "C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Managed"
+$(TargetFileName) -nokeypress</PostBuildEvent>
+  </PropertyGroup>
```

Accidentally left my build tasks in.
```suggestion
  <PropertyGroup>
  </PropertyGroup>
```